app/products/calls/state/actions.ts

```diff
@@ -566,3 +566,54 @@ export const setHost = (serverUrl: string, channelId: string, hostId: string) =>
     };
     setCurrentCall(nextCurrentCall);
 };
+
+export const processMeanOpinionScore = (mos: number) => {
+    const currentCall = getCurrentCall();
+    if (!currentCall) {
+        return;
+    }
+
+    if (mos < 4) {
```

Comment:
- I wonder if it would make sense to make the threshold a common constant in https://github.com/mattermost/calls-common 
- We should probably review the value before merging as I think 4 may be a little too aggressive. I can try to come up with something better but likely more towards 3.5.

Reply:
Good idea, done. Thanks!